generic/openshift/single-region/.tool-versions

```diff
@@ -0,0 +1 @@
+# Specific tools used by this implementation only
```

Comment:
just wondering should we have the file if it isn't really defined or only actually have it if ti differs from the default